### ISSUE TITLE
Redirect Request for Post-9/11 GI Bill Statement of Benefits tool

### ISSUE THREAD

**Issue by scjwalker** [department-of-veterans-affairs/va.gov-team#94306]
*Note from @scjwalker with CAIA: This url update is the result of an IA audit. So it needs doing, but it doesn't have dependancies with other work. I'm queueing up this redirect request ticket now, so that it's ready when the product team has a go-live date for the url change.*
**Update on 11/22: The team is ready to go live with the URL changes on Monday 11/25. The product team will take care of the redirects for the authenticated pages (Sam split those redirects into a separate ticket so that assignments are clear - it's [97680](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97680)), so only this one Drupal page redirect is needed from the PW team*

| POC | Github alias(es) |
--- | --- 
| Product team point(s) of contact: | @cassidy-beach  |
| IA point of contact: | @scjwalker |

### Type of request
- [x] We are changing the URL of an existing pages (complete redirect section)

### Do you have a product technical team able to implement this redirect?
- [x] No - CAIA will need to make Public Websites aware of the ticket.

### Implementation date
11/25

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/education/check-post-9-11-gi-bill-benefits/ | va.gov/education/check-remaining-post-9-11-gi-bill-benefits/

### Process, Roles and Responsibilities
- [x] Requesting team (IA): Above information is provided
- [ ] Requesting team (IA): All appropriate VA stakeholders are notified as appropriate - yes, EDM and Content have confirmed with Stakeholders
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team (PW): Work is complete
- [ ] Implementation team (PW): Validated in production
- [ ] Requesting team (IA): Validates everything is correct in production and closes ticket

**Comment by scjwalker** [comment]
@FranECross No need to pull this one in yet. I'm making the ticket now so that it's ready once the product team determines a date, but until then no action needed. I'll nudge you here once we've got a tentative date.

**Comment by FranECross** [comment]
@scjwalker Thanks so much for the great status at the top and the early awareness! 🎉

**Comment by scjwalker** [comment]
Added the "/print" url to the list of needed redirects. 
This need was surfaced by Taras from the product team while creating the new URL, and Fran confirmed that it was alright to add it to this ticket.

**Comment by scjwalker** [comment]
Note - The product team got stakeholder feedback that the stakeholders would like to use a different phrase than check-post-9-11-gi-bill-benefits here. I referred the ask to Content, since this ask is more at the scale of re-naming the tool than revising only the URL. Content and OCTO are going to discuss it.
I've asked @cassidy-beach to pause the product team's work on this url update, until we hear from Content and OCTO.

**Comment by FranECross** [comment]
@scjwalker I'll mark the ticket Blocked on our end until we hear back from you that it's a go. Thanks for the update!

**Comment by scjwalker** [comment]
Updated the new URLs we're changing to, based on work between the team, SMEs, and Content. Redirect ticket and [IA Spec](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/rename-post-9-11-GI-Bill-SoB.md) are consistent.

**Comment by FranECross** [comment]
@scjwalker Just doublechecking... are we awaiting this step before implementation? 'All appropriate VA stakeholders are notified as appropriate'  Thanks in advance!

**Comment by scjwalker** [comment]
My understanding @FranECross is that the product team should now be implementing this one instead. Is that correct?

**Comment by scjwalker** [comment]
In the time since this redirect was moved into Blocked status, CAIA and the team have been working on a re-name of the tool. This means that the URL of the unauthenticated Drupal page also needs to be updated now. I've added it as the top row in the Redirects table.

**Comment by scjwalker** [comment]
The EDM Team has confirmed that they are confident implementing the two redirects on the authenticated pages that they own. So:
- I have split those into a different redirect ticket [97680](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97680)
- PW now has only one Drupal page redirect to take care of, in this ticket.

**Comment by jilladams** [comment]
Removed Public Websites from this ticket, since product team is handling it.

**Comment by chriskim2311** [comment]
@scjwalker This redirect has made it to prod. Once you verify we can close this ticket thanks!

**Comment by scjwalker** [comment]
The redirect is working as intended in prod. Thank you for all your careful and speedy work on this, PW crew! 
Verified, and closing the ticket.
